### DISCUSSION TITLE
Multi-field editor "fields tab" missing.

### DISCUSSION THREAD

**Discussion by xmontero** [nocodb/nocodb#6670] [General]
Hi! This weekend I discovered NoCoDb and I fell in love with it.

Today I'm trying it for the first time.

I've seen here https://docs.nocodb.com/fields/multi-fields-editor/ that there's a field editor.

Nevertheless I don't see it: I only see Relations, APIs and Webhooks.

![image](https://github.com/nocodb/nocodb/assets/5641621/6d166408-7009-4e3f-ba41-3cd87c244841)

I'm running it locally in a docker in the SqlLite mode.

If I go to "data" there are 4 columns. I wanted to play around reordering the column order:

![image](https://github.com/nocodb/nocodb/assets/5641621/31ed9cc1-beb4-4d2c-830a-03de5817519e)

# Question

How do I access the multi-field editor?

**Comment by dstala** [comment]
@xmontero You need to upgrade to v0.202.4 (latest release)

**Reply by xmontero** [reply]
Humm... I just did a `docker run` against the `nocodb:latest` as stated in the first example here https://github.com/nocodb/nocodb#docker

How can I know what version am I running?

**Reply by xmontero** [reply]
Wait, I'm watching at the release history. https://github.com/nocodb/nocodb/tags

You mean the "field editor" was launched less than 2 hours ago?

**Reply by xmontero** [reply]
Indeed. I did a `docker pull nocodb/nocodb:latest` and here it is!!

Sorry for the inconvenience! This is state of the art, friends! I am in the european timezone, I just pulled nocodb before lunch, and in between lunch and the afternoon (just 2 hours, max 3 hours) a super-feature has been released hahahah

![image](https://github.com/nocodb/nocodb/assets/5641621/06716630-7263-4dd2-8eb2-1c1eb46d27ba)

Happy to join the community and willing to help if possible.

I am using this for a travel agency of my own. Will share ideas along the way!

Thanks @dstala for your prompt help!
